src/test/kotlin/com/karumi/maxibonkata/generators.kt

```diff
@@ -0,0 +1,20 @@
+package com.karumi.maxibonkata
+
+import io.kotlintest.properties.Gen
+
+object generators {
```

Comment:
I'm not sure if the usage of objects as namespaces is common in Kotlin. @Serchinastico is this implementation correct?

Reply:
`object` is the reserved word to create a singleton class in Kotlin. On that case, `generators` will be the name of the class and IMHO it should be capitalized. 
On the other hand, it seems that you could use functions to generate them without needed of the generators class because Kotlin allows you declaring Higher-Order functions

Reply:
I can only find the usages the kotlin docs provide here: https://kotlinlang.org/docs/reference/object-declarations.html#object-declarations

Even if we use them as namespaces they have the full potential of classes so I guess I'd go for CamelCase style.

Reply:
I meant CamelCase Style

Reply:
We use lowercase objects in scala but I guess it's not kotlin friendly :_( I've updated the PR changing ``generators`` to ``Generators``.